docs/CODE_GUIDELINES.md

```diff
@@ -407,9 +406,8 @@ import React, { useRef, MutableRefObject, RefObject } from 'react';
 
 ...
 
-const mutableRef: MutableRefObject<HTMLDivElement | null> = useRef<HTMLDivElement | null>(null);
+const ref = useRef() as MutableRefObject<HTMLInputElement>;
```

Comment:
thanks for updating the docs to match existing examples 👍 Can we remove these sentences from the paragraph at line 399 now?

> For mutable refs, use a union with `null` when initializing the ref; otherwise, TypeScript will complain when you change the value of `.current`. Similarly, *do not* create a union with `null` for non-mutable refs. 

also, are there use cases of the non-mutable `RefObject` currently?

Reply:
re: are there any non-mutable instances now? I don't know if there are or not